src/style-installer.js

```diff
@@ -300,10 +313,10 @@ function renderStartedNoInline(doc) {
 
 /**
  * Indicates that the body is always visible. For instance, in case of PWA.
- * @param {!Window} win
+ * @param {!Window} unusedWin
  */
-export function bodyAlwaysVisible(win) {
-  win[bodyVisibleSentinel] = true;
+export function bodyAlwaysVisible(unusedWin) {
```

Comment:
now I'm bit worried it will break PWA @choumx

Reply:
We can wait until we hear from Will.

Reply:
This might be fine for PWA but it won't work as expected across extensions (it's also used in `amp-access`). The problem is each JS binary will have its own version of this variable which is why we store services on the window.